spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -766,7 +766,7 @@ def parse_lsp_config(self, config):
         self.auto_completion_characters = (
             completion_options['triggerCharacters'])
         self.signature_completion_characters = (
-            signature_options['triggerCharacters'])
+            signature_options['triggerCharacters'] + ['='])  # FIXME:

```

Comment:
This is needed for this to work properly, but I dont know if this should be part of the dedault settings or something I should add here ?

Reply:
I don't understand very well where we need to set this. @andfoy, could you give us a hand here? Thanks.

Reply:
It took me a long time but I finally found where those options are set:

https://github.com/spyder-ide/spyder-pyls/blob/0.22.0/pyls/python_ls.py#L141-L143

So we should add `=` to them. But could you explain why is that needed?

Reply:
>But could you explain why is that needed?

Yes, of course. Take for example this code (the `*` represents the cursor)

```python
def somefunc(param1=0, param2=1, param3=None):
    pass

somefunc(param2=*
```

Without the `=` sign, when we are using a `parameter` that is located somewhere in the call, if we don't add the `=` it won't retrrigger a signature request, and the `parameter` highlighted will be incorrect. In the example `param2` will not be in red. Adding the `=` to the list of "trigger" characters solves this :-)

Reply:
Ok, thanks for digging that logic up! Could you submit a PR upstream:

https://github.com/palantir/python-language-server

explaining why that's needed? Thanks!

(Let's see if upstream accept it before adding it to our fork).

Reply:
See: https://github.com/palantir/python-language-server/pull/542

Reply:
Maybe you can make some noise @ccordoba12 ?